src/server/routes.ts

```diff
@@ -47,12 +56,30 @@ export const checkRoute = (path: string): RoutePath | Error => {
   return new Error("Invaild route path")
 }
 
-type AppRoutes = Record<
-  RoutePath,
+import Register from "../components/pages/register"
+
+const authRoutesDef = {
+  "/register/email": {
+    component: Register,
+    title: "Register with Email",
+  },
+}
+
```

Comment:
What's the reason for separating app routes from auth routes?

Reply:
2 main reasons:
- easier segregation behind feature flag
- likely the whole auth code will eventually get extracted out of the web wallet code and this should make it easier to separate.